sdk/lib/rtl/path.c

```diff
@@ -1672,6 +1672,36 @@ RtlSetCurrentDirectory_U(IN PUNICODE_STRING Path)
     return Status;
 }
 
+/*
+ * @implemented
+ */
+ULONG
+NTAPI
+RtlGetFullPathName_UEx(
+    _In_ PWSTR FileName,
+    _In_ ULONG BufferLength,
+    _Out_writes_bytes_(BufferLength) PWSTR Buffer,
+    _Out_opt_ PWSTR *FilePart,
+    _Out_opt_ RTL_PATH_TYPE *InputPathType)
+{
+    UNICODE_STRING FileNameString;
+
+    if (InputPathType)
+        *InputPathType = 0;
+
+    /* Build the string */
+    const NTSTATUS status = RtlInitUnicodeStringEx(&FileNameString, FileName);
```

Comment:
Please move this declaration above `if (InputPathType)` which for this reason TravisCI fails to build.

Reply:
We use: "list of variables -- code -- control-block (eg. if, while, for...) { list of variables -- code } -- code"
because we need to have the code compilable with e.g. MSVC 2010, i.e. in C90 standard.